### PR TITLE
Fix get event request

### PR DESCRIPTION
**Pull request by jbonnefont**
Fix the GetEvent service when it's use with graphdb

### CODE REVIEW

File: phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java [hunk 1]
```diff
@@ -268,80 +270,84 @@ private Event getEventFromBindingSet(BindingSet bindingSet) {
...
                 Oeev.concerns.getURI());
         Event event = null;
-        
+
         // Get event from storage
         TupleQuery eventsTupleQuery = getConnection().prepareTupleQuery(QueryLanguage.SPARQL, eventQuery.toString());
-        
+
         try {
             TupleQueryResult eventsResult = eventsTupleQuery.evaluate();
```
Comment (annetireau) [inline phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java]: I guess it's the same problem

File: phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java [hunk 2]
```diff
@@ -584,17 +590,17 @@ private SPARQLQueryBuilder prepareCountQuery(String searchUri, String searchType
...
+                searchType,
+                searchConcernedItemLabel,
+                searchConcernedItemUri,
+                dateRangeStartString,
                 dateRangeEndString);
-        
+
         TupleQuery tupleQuery = getConnection().prepareTupleQuery(QueryLanguage.SPARQL, countQuery.toString());
         Integer count = 0;
         try (TupleQueryResult result = tupleQuery.evaluate()) {
```
Comment (annetireau) [inline phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java]: And also here

File: phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java [hunk 3]
```diff
@@ -268,80 +270,84 @@ private Event getEventFromBindingSet(BindingSet bindingSet) {
...
-                Contexts.EVENTS.toString(), 
+                user,
+                Contexts.EVENTS.toString(),
                 Oeev.concerns.getURI());
-        
+
         // for each event, set its properties and concerned Items
         try {
             TupleQueryResult eventsResult = eventsTupleQuery.evaluate();
             while (eventsResult.hasNext()) {
```
Comment (annetireau) [inline phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java]: What is the return of .next() ? A NoSuchElementException ?
If yes maybe you could change the While test.

Comment (jbonnefont) [reply phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java]: With GraphDB it returns a BindingSet of type EmptyBindingSet with all values to null and size to 0..

File: phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java (hunk 2 above)
Comment (jbonnefont) [reply phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java]: If there are no EventDAO created, it don't go there

File: phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java (hunk 1 above)
Comment (jbonnefont) [reply phis2-ws/src/main/java/opensilex/service/dao/EventDAO.java]: if you are searching an Event with a wrong Uri there are no results, and eventsResult.hasNext() is false

**Pull request merged**